packages/generator/src/operations/operation.ts

```diff
@@ -45,6 +45,8 @@ export function operationFunctionBase(
       [
         getDocDescription(operation),
         `@param parameters - Object containing all parameters for the ${operation.type}.`,
+        // TODO: is this still necessary, considering that the line above indicates the type of operation?
+        `@type: ${operation.type}`,
```

Comment:
[req] This is not only not necessary, but even wrong, because in TSDoc this tag (`@type`) does not exist and in JSDoc it represents an actual type => https://jsdoc.app/tags-type.html

Reply:
Thanks, I'll remove this then.